src/main/kotlin/lottery/domain/LottoGame.kt

```diff
@@ -0,0 +1,21 @@
+package lottery.domain
+
+class LottoGame(val price: Int, numberGenerator: LottoNumberGenerator = RandomNumberGenerator()) {
+    private val lottos: Lottos
+
+    init {
+        lottos = Lottos.of(price.div(LOTTERY_PRICE), numberGenerator)
+    }
```

Comment:
[secondary constructor](https://kotlinlang.org/docs/classes.html#secondary-constructors)를 활용하면 다음과 같이 작성할 수도 있겠네요!
```suggestion
class LottoGame(val price: Int, private val lottos: Lottos) {

    constructor(price: Int, numberGenerator: LottoNumberGenerator = RandomNumberGenerator()) : this(price, Lottos.of(price.div(LOTTERY_PRICE), numberGenerator))
```
참고만 해주셔도 좋습니다 🙂

Reply:
부생성자 기능을 사용하니 `numberGenerator` 객체가 LottoGame 생성시에만 사용되는군요!